react/MuiCozyTheme/ExpansionPanel/Readme.md

```diff
@@ -0,0 +1,33 @@
+```
+const MuiCozyTheme = require('..').default
```

Comment:
Why are you using `require()` instead of `import`?

Reply:
Actually just because I copy/pasted the `Button` readme, and it uses `require`. I just tested with `import` and got an error:

```
CompileError: import is not supported (1:0)
1 : import MuiCozyTheme from '../index.js'
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

It seems styleguidist doesn't support `import` by default.

Reply:
well, there an import later in `react/MuiCozyTheme/ExpansionPanel/index.js`. Is it because you looked for index.js and not index.jsx ?

Reply:
It does not support `import` inside the examples.